radicle/src/test/arbitrary.rs

```diff
@@ -60,9 +60,11 @@ pub fn vec<T: Eq + Arbitrary>(size: usize) -> Vec<T> {
 
 pub fn nonempty_storage(size: usize) -> MockStorage {
     let mut storage = gen::<MockStorage>(size);
-    storage
-        .inventory
-        .insert(gen::<Id>(size), gen::<Doc<Verified>>(size));
+    for _ in 0..size {
+        storage
+            .inventory
+            .insert(gen::<Id>(size), gen::<Doc<Verified>>(size));
```

Comment:
```suggestion
            .insert(gen::<Id>(1), gen::<Doc<Verified>>(1));
```
The size parameter doesn't have any effect on things with a fixed size, such as ids or docs, so we usually set it to `1`